### ISSUE TITLE
Provide an option to skip installing other cni binaries even when absent

### ISSUE THREAD

**Issue by abhinavdahiya** [projectcalico/cni-plugin#339]
## Current Behavior

https://github.com/projectcalico/cni-plugin/blob/master/k8s-install/scripts/install-cni.sh#L56-L64
script installs other cni binaries when they are absent.

## Possible Solution

It would be great if there option to disable copying binaries even if they were not present. like:
```bash
INSTALL_OTHER_CNI_BINARIES=${INSTALL_OTHER_CNI_BINARIES:-false}

if [[ ${INSTALL_OTHER_CNI_BINARIES} = "true" && ! -f /host/opt/cni/bin/flannel ]]; then
cp /opt/cni/bin/flannel /host/opt/cni/bin/
fi
```
## Context

For example I am trying to use flannel+calico and want flannel manifest to copy its binary version to correct directory. There might be conflicts depending on who executes first.

**Comment by caseydavenport** [comment]
This sounds useful to me.

Probably on a per-plugin basis? e.g. you might want host-local but not flannel?

**Comment by abhinavdahiya** [comment]
@caseydavenport So there would be INSTALL_FLANNEL_CNI_BINARY, INSTALL_HOSTLOCAL_CNI_BINARY and INSTALL_LOOPBACK_CNI_BINARY options ??

Not sure if individual options would be necessary. But if it doesn't create too much complexity it would be nice.

**Comment by caseydavenport** [comment]
Yeah, separate options would work fine.  Or maybe a single one with multiple clauses?

e.g.

`INSTALL_BINARIES=calico,calico-ipam,flannel,host-local,loopback,portmap`

Or,

`SKIP_BINARIES=flannel,loopback`

**Comment by tmjd** [comment]
Fixed by #341
